packages/lokse/src/core/reader/__tests__/worksheet-reader.test.ts

```diff
@@ -7,7 +7,7 @@ const createWorksheet = (title: string, index = 1) => ({
 });
 
 describe("WorksheetReader.shouldUseWorksheet", () => {
-  it("should use worksheet when empty or null or start", () => {
+  it("should use worksheet when empty or null or asterisk", () => {
```

Comment:
or obelisk :trollface:

Reply:
😁
https://en.wikipedia.org/wiki/Asterisk here you are you little troll 😄